src/main/configurations/morcoreMessageProcessor/Configuration_FromMorCore.xml

```diff
@@ -4,7 +4,7 @@
         <Receiver name="Frank_ListTaakTypes">
             <ApiListener
                 name="Frank_ListTaakTypes"
-                uriPattern="/v1/taaktypes"
+                uriPattern="/v1/taaktype"
```

Comment:
Needs to be /api/v1/taaktype. The "/api" section was added in a recent update. Make sure your branch is up-to-date.